### ISSUE TITLE
Is backward_G_GAN_content correct?

### ISSUE THREAD

**Issue by kekke-gk** [HsinYingLee/DRIT#30]
Thank you for sharing your source code.
At line 343, loss value is not added to `ad_loss`. Is this correct?

I think you should add loss values to `ad_loss` like this.
`ad_loss += nn.functional.binary_cross_entropy(outputs_fake, all_half)`

https://github.com/HsinYingLee/DRIT/blob/dbfa8043bb9cfa79170c7b6c3afd1fe6f7e4ebe9/src/model.py#L338-L344

**Comment by HsinYingLee** [comment]
Yes it should be "+=". But since "outs" only has one element, the computation will be the same in this case.

**Comment by kekke-gk** [comment]
I understand. Thank you.
